### PR TITLE
modified the view

### LINKED ISSUE #6: Modify the view
add a temporary item list to use for JS.

### PR DESCRIPTION
**Pull request by tahminarasoli**
I have done some changes but there is a problem in the small red button view. please have a look if you can help with that. maybe it is not so difficult but i prefared to share with you than bieng bussy with it by my self. after fixed this will close #6

### CODE REVIEW

**Comment by bermarte** [comment]
I am just reading it.

**Comment by bermarte** [comment]
@tahminarasoli  Let me check it, I will be back to you.

**Comment by bermarte** [comment]
@tahminarasoli Can you describe the problem? Not sure if I understand.

**Comment by tahminarasoli** [comment]
Capture
I want the button at the end of table row.

**Comment by bermarte** [comment]
Ok, let me try.

**Comment by bermarte** [comment]
Add class='float-right' to the td where the button is

**Comment by bermarte** [comment]
I would change the color of the text when you hover on the todo, it should be white or lighter.

File: public/index.html [hunk 1]
```diff
@@ -34,13 +36,36 @@ <h2>
...
+                <th>Title </th>
+                <th>Description</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody id="todo-list">
+              <tr>
+                <td>Title </td>
+                <td>Description</td>
+                <td><a href="#" class="btn btn-danger btn-sm delete">X</a></td>
```
Comment (bermarte) [inline public/index.html:61]: add class='float-right'

**Review by bermarte — CHANGES_REQUESTED** [review]

**Review by bermarte — APPROVED** [review]
Normally in bootstrap, if you want to add the class, you do it like that
<td class='float-right'>...</td>
because the class is already written for you to be used

**Pull request merged**